identity/webapp/pages/api/registration-form.ts

```diff
@@ -1,27 +1,47 @@
 import type { NextApiRequest, NextApiResponse } from 'next';
 import { generateNewToken, decodeToken } from '../../src/utility/jwt-codec';
+import axios from 'axios';
+import getConfig from 'next/config';
+
+const { serverRuntimeConfig: config } = getConfig();
 
 export default (req: NextApiRequest, res: NextApiResponse): void => {
   if (req.method === 'POST') {
-    const {
-      state,
-      redirectUri,
-      firstName,
-      lastName,
-      termsAndConditions,
-      sessionToken,
-    } = req.body;
+    const { state, firstName, lastName, termsAndConditions, sessionToken } =
+      req.body;
+
+    if (
+      !state ||
+      !firstName ||
+      !lastName ||
+      !termsAndConditions ||
+      !sessionToken
+    ) {
+      res.status(400).json({
+        error: 'Missing required fields',
+      });
+      return;
+    }
 
     const decodedToken = decodeToken(sessionToken);
```

Comment:
What if this throws? Something to bikeshed as to whether that's a 401 or a 400

Reply:
<img width="338" alt="image" src="https://user-images.githubusercontent.com/1394592/165089058-40a7506e-c3da-4060-82cc-13f94098084e.png">
 co-pilot reckons 400 so I guess that settles it?